src/main/webapp/app/shared/exercise-filter/exercise-filter-modal.component.scss

```diff
@@ -0,0 +1,14 @@
+::ng-deep ngb-typeahead-window.dropdown-menu {
+    max-height: 25rem !important;
+    overflow-y: auto !important;
+}
+
+.no-left-margin-padding:first-child {
+    margin-left: 0 !important;
+    padding-left: 0 !important;
+}
+
+.form-control:disabled,
+.form-control[disabled] {
+    pointer-events: none;
```

Comment:
At least not in this case; here we need to overwrite it as otherwise, the dropdown will change the color on hover, even if no further selections are possible.
I added a comment for an explanation!

![ExerciseFilterReview-HoverEffect](https://github.com/user-attachments/assets/a5ffd20c-4f44-44fa-abc3-e5baa852a7ae)